apps/chart-docsite/.eslintrc.json

```diff
@@ -4,7 +4,7 @@
   "rules": {
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33527&commitId=05eae6ad8fa0a83258a3a24ce097f08b80c89db3&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|7| Changed|
|Avatar Converged.badgeMask.chromium.png|24| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|2825| Changed|
|Drawer.Full Overlay RTL.chromium.png|1157| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>